project/test_project/src/main.c

```diff
@@ -1,56 +0,0 @@
-// Test program for STM32F072RB discovery boards - attempts to connect to the onboard MEMS
```

Comment:
If you need some help on how to do this, you can follow these instructions. First find the commit that still contains the ``project/test_project/src/main.c`` file by looking through the commit history.

```bash
git log -- project/test_project/src/main.c
```

Look for the commit before the one where you made your changes. This gives you commit hash ``67760ab66a3300740d52ab3f1c61797933d75a01``.

Then checkout the file revision at that commit.

```bash
git checkout 67760ab66a3300740d52ab3f1c61797933d75a01 project/test_project/src/main.c
```